src/routes/group.routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express'
+
+import { isAuthenticated } from '../config/passport'
+import { create, createForm, destroy, index, join, leave, show } from '../controllers/group'
+import { isGroupOwner } from '../middlewares/group.middleware'
+
+const router = Router()
+
+router.get('/', isAuthenticated, index)
+router.get('/new', isAuthenticated, createForm)
+router.post('/new', isAuthenticated, create)
+router.get('/:id', isAuthenticated, show)
+router.post('/:id/join', isAuthenticated, join)
+router.post('/:id/leave', isAuthenticated, leave)
+router.post('/:id/delete', isAuthenticated, isGroupOwner, destroy)
```

Comment:
I've just seen it and it is out of the scope of this PR, but at some point it'd nice to change it to `delete` instead of a `post` method.

Reply:
Yep, was planning to do so.